src/components/vr_cooperation/include/vr_cooperation/commands/base_command_request.h

```diff
@@ -165,11 +172,10 @@ class BaseCommandRequest : public Command,
   virtual void Execute() = 0;
 
   application_manager::MessagePtr message_;
+  VRModule* parent_;
```

Comment:
In RegisterServriceRequest class, I have to check if VR Service is suported or not. To do this, I have to call VRModule's fucntion in RegisterServriceRequest .
So I made VRModule* parent_ protected